pygmt/tests/test_histogram.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=redefined-outer-name
+"""
+Tests histogram.
+"""
+import pytest
+from pygmt import Figure
+
+
+@pytest.fixture(scope="module")
+def table():
+    """
+    Returns a list of integers to be used in the histogram.
+    """
+    return [1, 1, 1, 1, 1, 1, 2, 2, 2, 3, 4, 5, 6, 7, 8, 8, 8, 8, 8, 8]
```

Comment:
![image](https://user-images.githubusercontent.com/3974108/115970027-eb8d5c80-a50d-11eb-97d9-a4106687a6b6.png)

Is the baseline image correct? I expect to see y=6 for x=8.

BTW, please change `projection` to a smaller dimension like `X10c/5c`.

Reply:
@seisman I just tried it on both GMT 6.1.1 and 6.2.0rc1, with the figure coming out the same way. Any idea what might be causing the miscount for 8?

Also, changing the projection to `X3c/10c` to give it a larger vertical scale; does that work?

Reply:
It looks like an upstream bug.

Reply:
I'll try and recreate it in GMT and open an upstream issue

Reply:
GMT automatically determines the x range from input data. In this example, xmin=1, xmax=8, but the vertical bar for x=8 is outside of the [1, 8] range, so it's not plotted.

You should be able to see the x=8 bar, if you manually set `region=[0, 9, 0, 6]`. I think we can do it temporarily so that this PR can be merged, and report it to upstream and wait for feedback and potential fix.

Reply:
> Also, changing the projection to `X3c/10c` to give it a larger vertical scale; does that work?

BTW, the large vertical scale makes the image difficult to read, why not just use "X10c/10c" for a square basemap?

Reply:
@seisman I made your suggested changes